pinot-query-runtime/src/main/java/org/apache/pinot/query/service/dispatch/QueryDispatcher.java

```diff
@@ -303,7 +303,7 @@ Map<String, String> initializeTimeSeriesMetadataMap(TimeSeriesDispatchablePlan d
     Map<String, String> result = new HashMap<>();
     result.put(WorkerRequestMetadataKeys.LANGUAGE, dispatchablePlan.getLanguage());
     result.put(WorkerRequestMetadataKeys.START_TIME_SECONDS,
-        Long.toString(dispatchablePlan.getTimeBuckets().getStartTime()));
+        Long.toString(dispatchablePlan.getTimeBuckets().getTimeBuckets()[0]));
```

Comment:
nit: consider creating a variable for `dispatchablePlan.getTimeBuckets()` to reuse here and in the following lines.